client/src/main/java/io/pravega/client/admin/impl/ReaderGroupManagerImpl.java

```diff
@@ -111,7 +122,19 @@ public void deleteReaderGroup(String groupName) {
                                              throw Exceptions.sneakyThrow(e);
                                          }),
                                RuntimeException::new);
-        
+
+        AsyncIterator<Stream> streamIter = controller.listStreams(scope);
+        while (streamIter.getNext() != null) {
+            String streamName = streamIter.getNext().join().getStreamName();
+            controller.listSubscribers(scope, streamName)
```

Comment:
Using `controller.listSubscribers()` does not look right.
We can get the list of Streams from ReaderGroupConfig and call deleteSubscriber on each.

Reply:
I did this because I wasn't sure how to call the ReaderGroupConfig for  a given ReaderGroup name.

Reply:
Made changes to use the ReaderGroupConfig instead.

Reply:
Suggestion noted. Instead of using `controller.listSubscribers()`, the list of streams is obtained from the `ReaderGroupConfig`.